config/default/GcpTokenCacheItemPool.conf.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
true, it is why I think we should install it only when needed, via seed file and only on spanner env

Reply:
What do you suggest? To delete the `.conf.php` file? I think we will have bugs in the platform. I would like to avoid another 1h installation to check it. Even if this file is there it will not be used, so I do not see a risk. Also, when we have a class with `SERVICE_ID` we by standard create a conf.php. It would be strange in this case to not do it.

@mike-ionut-mihai-sandu-tao @bartlomiejmarszal @mike-ionut-mihai-sandu-tao Can you guys give some input on this, please?

Reply:
For example, on AWS environment, and only on this kind of env, we have `awsClient.conf.php`

Reply:
if we can leave out the initial default config and put this inside the readme then ok, but overall some registration of that information will be there anyway ,  so i don't consider this an impediment to close the pr
- if we strictly refer to the fact that we are adding 1 more class and that this will increase maintenance i agree
- if we refer that usually the learning curve  is a bit bigger and is hard to find configs and the fact that documentation is not proper i agree also
so from my point of view : 
 - let's leave the default config or make 100% sure that this is easy to configure and that the information is reachable

Reply:
For me it is just a config there that will only be used if we call the SpannerDriver, otherwise should not cause any issue. The conf.php is basically our "DI" mapping. You do not remove things from DI mapping just because you are not using it. So IMO we should maintain.

Reply:
Discussed in person with @siwane . I removed the migrations file with this.